### DISCUSSION TITLE
How do I use a toggle like this page?

### DISCUSSION THREAD

**Discussion by Irisation23** [community/community#41346] [Discussions] [answered]
### Select Topic Area

Question

### 👋 hellowww ladies and gentlemens!
- I want to use the toggle in the discussion category.
- How can you do that?
![image](https://user-images.githubusercontent.com/53285909/206863960-5e23453b-713e-4900-ac7b-fd8645659871.png)
  - I want to use the function in the discussion like the picture above.

Thank you in advance 🙏🏻

**Comment by DSDmark** [comment, marked as answer]
Hi @bunsung92,

You mean this:-

<details>
<summary>Product feedback</summary>

:+1: Accessibility
:ship: Action Packages
:arrows_counterclockwise:  API  

</details>

here is the code:-
```md
<details>
<summary>Product feedback</summary>

:+1: Accessibility

:ship: Action Packages

:arrows_counterclockwise:  API  

</details>
```

**Reply by Irisation23** [reply]
thx @DSDmark  
Can the code be applied as a GUI?
If GUI is not possible, can you tell me how to apply the code? plz help me 😿

**Reply by DSDmark** [reply]
@bunsung92, what do you mean by **GUI**. It's a Markdown code.

**Reply by Irisation23** [reply]
@DSDmark, I think my question is vague 
![image](https://user-images.githubusercontent.com/53285909/206911671-40c487a2-02d5-4a36-9e7c-94c2b8b6a827.png)
- this is my discussion Categories

![image](https://user-images.githubusercontent.com/53285909/206911794-13713853-19ab-4d42-bb25-a32305303148.png)
- And i want discussion Categories looks like this Product feedback
- Is there a way for a category to have **depth**?

**Reply by DSDmark** [reply]
This is generated by **chartGPT**.

> To create a drop-down menu of discussion categories in GitHub, you would need to use a custom script or plugin. GitHub does not have native support for creating drop-down menus in discussions.
One way to create a drop-down menu of discussion categories is to use a static site generator like Jekyll or Hugo to create your GitHub pages. These tools allow you to create templates for your pages and include custom scripts or plugins to create the drop-down menu.
Another option is to use a third-party plugin or script that adds support for drop-down menus to your GitHub pages. There are several plugins and scripts available that can be easily added to your pages to create a drop-down menu of discussion categories.
Overall, creating a drop-down menu of discussion categories in GitHub requires some custom coding or the use of a third-party plugin. It is not a feature that is natively supported by GitHub.

**Reply by Irisation23** [reply]
My curiosity has been solved. And this answer really helped a lot.
I'll apply it as you told me. Thanks once again. Have a nice day.
